### PR TITLE
Add Service Directory GKE Integration recipes

### PR DESCRIPTION
**Pull request by katiehochberg**

### CODE REVIEW

**Comment by katiehochberg** [comment]
cc @mark-church

File: service-directory/cluster-ip-service/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Service Directory GKE Integration - ClusterIP Service
+
+Service Directory for GKE is a cloud-hosted controller for GKE Clusters that
+sync services to Service Directory.
```
Comment (boredabdel) [inline service-directory/cluster-ip-service/README.md:4]: Can you please capitalize the S in service to make sure it points to Kubernetes Services.

**Review by boredabdel — COMMENTED** [review]
One comment to fix.

Also could you please add a README.md file under the service-directory folder with sublinks to the examples.

Thanks for this PR

File: service-directory/cluster-ip-service/README.md (hunk 1 above)
Comment (katiehochberg) [reply service-directory/cluster-ip-service/README.md]: I've fixed all of the capitalization issues in the README, and added a main README for the folder that points to all of the examples.

**Review by boredabdel — APPROVED** [review]
LGTM. Thanks for this PR

**Pull request merged**